bundles/org.openhab.binding.linky/src/main/java/org/openhab/binding/linky/internal/api/EnedisHttpApi.java

```diff
@@ -128,13 +132,13 @@ public void initialize() throws LinkyException {
             }
 
             authData.callbacks.get(1).input.get(0).value = config.password;
-            logger.debug("Step 4 : auth2 - send the auth data");
-            result = httpClient.POST(authenticateUrl).header(HttpHeader.CONTENT_TYPE, "application/json")
+            logger.debug("Step 4: auth2 - send the auth data");
+            result = httpClient.POST(authenticateUrl).header(HttpHeader.CONTENT_TYPE, MediaType.APPLICATION_JSON)
```

Comment:
I'm not sure if this could cause more than one `Content-Type` header, which could be rejected by some webservers. The safer approach might be to provide the content-type below instead of manually adding the header:
```java
.content(new StringContentProvider(gson.toJson(authData), MediaType.APPLICATION_JSON))
```

Reply:
Thank you for your suggestion but the binding does not break at this step. To be sure I tested your proposal (that seems interesting) but it also breaks authentication process.